vscode/src/client/analyzerClient.ts

```diff
@@ -50,34 +51,69 @@ export class AnalyzerClient {
       mutateExtensionState((draft) => {
         draft.isFetchingSolution = flag;
       });
-    this.outputChannel = vscode.window.createOutputChannel("Konveyor-Analyzer");
+
+    this.outputChannel = vscode.window.createOutputChannel("Konveyor-Analyzer"); //, { log: true });
+
+    this.assetPaths = buildAssetPaths(extContext);
     this.kaiDir = path.join(buildDataFolderPath()!, "kai");
     this.kaiConfigToml = path.join(this.kaiDir, "kai-config.toml");
+
+    this.outputChannel.appendLine(
+      `current asset paths: ${JSON.stringify(this.assetPaths, null, 2)}`,
+    );
+    this.outputChannel.appendLine(`Kai directory: ${this.kaiDir}`);
+    this.outputChannel.appendLine(`Kai config toml: ${this.kaiConfigToml}`);
   }
 
   public async start(): Promise<void> {
     if (!this.canAnalyze()) {
-      vscode.window.showErrorMessage("Cannot start the server due to missing configuration.");
+      vscode.window.showErrorMessage(
+        "Cannot start the kai rpc server due to missing configuration.",
+      );
       return;
     }
 
+    // TODO: If the server generates files in cwd, we should set this to something else
+    const serverCwd = this.extContext.extensionPath;
+
+    // TODO: How different does the server's env need to be from vscode's env?
+    const serverEnv = {
+      ...process.env,
+      GENAI_KEY: "BWAHAHA",
```

Comment:
This is probably the time to remove this. Looking in kai it appears they already have dummy values for when it's unset, presumably for dev environment purposes.

Reply:
I'll spin out a function as a placeholder for if/when we need to patch env vars together for the kai rpc server.